File: server/src/main/java/com/aifriend/question/application/QuestionAnsweringPort.java
package com.aifriend.question.application;

/**
 * 生活问答端口。
 *
 * <p>当前仅占位，不注册实现、不开放 API，不调用云模型或外部搜索。
 *
 * @author Codex
 * @since 1.0.0
 */
public interface QuestionAnsweringPort {

    /**
     * 生成带来源的问答草稿。
     *
     * @param question 去除敏感信息后的问题
     * @return 问答草稿
     */
    String prepareAnswer(String question);
}

File: android/app/src/main/java/com/aifriend/core/question/QuestionAnsweringPort.kt
package com.aifriend.core.question

/**
 * 生活问答端口。
 *
 * 当前只占位，不调用云模型、外部搜索或第三方服务。
 */
interface QuestionAnsweringPort {
    /** 生成带来源的问答草稿。 */
    suspend fun prepareAnswer(question: String): String
}
